fedora/client/openidbaseclient.py

```diff
@@ -182,7 +183,8 @@ def _authed_get(self, url, params=None, data=None, **kwargs):
         response = self._session.get(url, params=params, data=data, **kwargs)
         return response
 
-    def send_request(self, method, auth=False, verb='POST', **kwargs):
+    def send_request(self, method, auth=False, verb='POST', retries=None,
+                     retry_sleep=0.5, **kwargs):
```

Comment:
OK, so now you should kick it out of the `send_request()` parameters and use `self.retry_sleep` inside it instead.